run_bot.sh

```
@@ -616,7 +650,7 @@ run_with_json() {
     # List generated files
     if [ -d "$output_dir" ] && [ "$(ls -A $output_dir)" ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)